cnf-tests/testsuites/e2esuite/multinetworkpolicy/multinetworkpolicy_sriov.go

```diff
@@ -0,0 +1,819 @@
+			eventually30s(nsZ_podC).ShouldNot(Reach(nsX_podA))
+
+			eventually30s(nsX_podB).ShouldNot(Reach(nsX_podA))
+			eventually30s(nsX_podC).ShouldNot(Reach(nsX_podA))
+		})
+
+		It("ALLOW traffic to nsX_podA only from (nsY/* OR */podB)", func() {
+
+			MakeMultiNetworkPolicy(TEST_NETWORK,
+				WithPodSelector(metav1.LabelSelector{
+					MatchLabels: map[string]string{
+						"pod": "a",
+					},
+				}),
+				WithIngressRule(multinetpolicyv1.MultiNetworkPolicyIngressRule{
+					From: []multinetpolicyv1.MultiNetworkPolicyPeer{
+						{
+							NamespaceSelector: &metav1.LabelSelector{
+								MatchLabels: map[string]string{
+									"kubernetes.io/metadata.name": nsY,
+								},
+							},
+						},
+						{
+							PodSelector: &metav1.LabelSelector{
+								MatchLabels: map[string]string{
+									"pod": "b",
+								},
+							},
+						},
+					},
+				}),
+				CreateInNamespace(nsX),
+			)
+
+			// Allowed
+			eventually30s(nsY_podA).Should(Reach(nsX_podA))
+			eventually30s(nsY_podB).Should(Reach(nsX_podA))
+			eventually30s(nsY_podC).Should(Reach(nsX_podA))
+
+			eventually30s(nsZ_podB).Should(Reach(nsX_podA))
+			eventually30s(nsX_podB).Should(Reach(nsX_podA))
+
+			// Not allowed
+			eventually30s(nsZ_podA).ShouldNot(Reach(nsX_podA))
+			eventually30s(nsZ_podC).ShouldNot(Reach(nsX_podA))
+
+			eventually30s(nsX_podC).ShouldNot(Reach(nsX_podA))
+		})
+
+		It("ALLOW traffic to nsX_podA only from (namespace IN {nsY, nsZ} AND pod IN {podB, podC})", func() {
+
+			Skip("LabelSelectorwith multiple In values is not yet supported by multi-networkpolicy-iptables - https://github.com/k8snetworkplumbingwg/multi-networkpolicy/issues/4")
+			//	E0511 14:37:07.698115       1 policyrules.go:238] pod selector: operator "In" without a single value cannot be converted into the old label selector format
+
+			MakeMultiNetworkPolicy(TEST_NETWORK,
+				WithPodSelector(metav1.LabelSelector{
+					MatchLabels: map[string]string{
+						"pod": "a",
+					},
+				}),
+				WithIngressRule(multinetpolicyv1.MultiNetworkPolicyIngressRule{
+					From: []multinetpolicyv1.MultiNetworkPolicyPeer{{
+						NamespaceSelector: &metav1.LabelSelector{
+							MatchExpressions: []metav1.LabelSelectorRequirement{{
+								Key:      "kubernetes.io/metadata.name",
+								Operator: metav1.LabelSelectorOpIn,
+								Values:   []string{nsY, nsZ},
+							}},
+						},
+						PodSelector: &metav1.LabelSelector{
+							MatchExpressions: []metav1.LabelSelectorRequirement{{
+								Key:      "pod",
+								Operator: metav1.LabelSelectorOpIn,
+								Values:   []string{"b", "c"},
+							}},
+						},
+					}},
+				}),
+				CreateInNamespace(nsX),
+			)
+
+			// Allowed
+			eventually30s(nsY_podB).Should(Reach(nsX_podA))
+			eventually30s(nsY_podC).Should(Reach(nsX_podA))
+			eventually30s(nsZ_podB).Should(Reach(nsX_podA))
+			eventually30s(nsZ_podC).Should(Reach(nsX_podA))
+
+			// Not allowed
+			eventually30s(nsX_podA).ShouldNot(Reach(nsX_podA))
```

Comment:
Would we like to check that the pod can reach itself? or it's a typo?

Reply:
that's  a typo, good catch.

From regular [NetworkPolicy doc](https://kubernetes.io/docs/concepts/services-networking/network-policies/)

> Other pods that are allowed (exception: a pod cannot block access to itself)

Moving to 
```
eventually30s(nsX_podA).Should(Reach(nsX_podA))
```
to ensure that behavior. Note that this test is skipped,waiting for:
- https://github.com/k8snetworkplumbingwg/multi-networkpolicy/issues/4